tools/vscode/ddlog-language/src/good/base.json5

```diff
@@ -38,7 +38,13 @@
       },
       { 
         "include": "#storage"
-      }
+      },
```

Comment:
won't this apply the same highlighting rules for .dl and .dat files?
Is there a way to have separate rules for .dl which are distinct from rules for .dat?